configure

```diff
@@ -0,0 +1,363 @@
+#!/bin/sh
+### Custom configure script of Dinit.
+## Initial prepartion
+set -eu
+cd "$(dirname "$0")"
+
+## Helper functions
+# According to POSIX, echo has unspecified behavior if any of its arguments contain “\” or
+# if its first argument is “-n”. Unix™ standards fill in this unspecified area for
+# XSI-conformant implementations, by specifying nasty undesirable behavior that no one wants
+# (“\” is interpreted as a C-string-literal style escape), and other popular implementations
+# such as Bash interpret argument values other than “-n” as special options even
+# when in “POSIX compatibility” mode, rendering then nonconformant.
+# These issues mean echo "$var" can sting you whenever you do not have strict guarantees
+# about the contents of var (for example that it contains a nonnegative integer).
+# So we replace shell built-in echo with a printf based function.
+# For more info see http://www.etalabs.net/sh_tricks.html
+echo()
+{
+    IFS=" " printf %s\\n "$*"
+}
+
+info()
+{
+    echo "$*"
+}
+
+sub_info()
+{
+    echo "  ... $*"
+}
+
+error()
+{
+    echo "Error: $*"
+    exit 1
+}
+
+sub_error()
+{
+    echo "  ... Error: $*"
+    exit 1
+}
+
+warning()
+{
+    echo
+    echo "Warning: $*"
+}
+
+sub_warning()
+{
+    echo "  ... $*"
+    echo
+}
+
+create_testfile_cc()
+{
+    echo "int main(int argc, char **argv) { return 0; }" > testfile.cc
+}
+
+findcxx()
+{
+    if type "$1" > /dev/null 2>&1; then
+        if [ "${2:-}" = "host" ]; then
```

Comment:
This function is never used with `host` as an argument, I think you can remove this check